scripts/install.sh

```diff
@@ -13,14 +13,12 @@ if [ "$arch" = "arm64" ]; then arch="aarch64"; elif [ "$arch" = "x86_64" ]; then
 if [ "$os" = "darwin" ]; then
     os_pattern="apple-darwin"
 else
-    os_pattern=$os
+    os_pattern="unknown-linux-gnu"
```

Comment:
that's going to fail for BSDs but we don't really talk about them in the install docs so it's probably ok